<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IssueReporter</class>
 <widget class="QDialog" name="IssueReporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report an Issue</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
   <property name="spacing">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Consider including the log for the session that had an issue, if applicable.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_logs_view">
       <property name="text">
        <string>View Logs:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CComboBox" name="comboBox_sessions"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="LogViewer" name="log_viewer">
     <property name="plainText">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Log file is at:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_log_path">
       <property name="text">
        <string notr="true">&lt;log file path&gt;</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_name_hidden">
       <property name="text">
        <string notr="true">&lt;name was hidden&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_clipboard">
       <property name="text">
        <string>Copy to Clipboard</string>
       </property>
       <property name="icon">
        <iconset theme="edit-copy">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_open_issues">
       <property name="text">
        <string>Open Issue Tracker</string>
       </property>
       <property name="icon">
        <iconset theme="internet-services">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset theme="window-close">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CComboBox</class>
   <extends>QComboBox</extends>
   <header>deepqt.CustomQ.CComboBox</header>
  </customwidget>
  <customwidget>
   <class>LogViewer</class>
   <extends>QPlainTextEdit</extends>
   <header>pcleaner.gui.log_viewer</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
